solved_exercises/dani/02-tuples.py

```diff
@@ -1,14 +1,16 @@
 import itertools
 '************************************************'
```

Comment:
now, let's adopt the pythonistic way of writing code: simple and clean.
remove the dummy docstrings, they do not add any value.

Reply:
bun, schimb abordarea cu #, dar tot vreau sa vad o delimitare in pagina intre exercitii, e piticu meu :)